packages/BetterPhpDocParser/PhpDocParser/BetterPhpDocParser.php

```diff
@@ -40,7 +40,25 @@ public function __construct(
         private readonly iterable $phpDocNodeDecorators,
         private readonly PrivatesAccessor $privatesAccessor = new PrivatesAccessor(),
     ) {
-        parent::__construct($typeParser, $constExprParser);
+        parent::__construct(
+            // TypeParser
+            $typeParser,
+            // ConstExprParser
+            $constExprParser,
+            // requireWhitespaceBeforeDescription
+            false,
+            // preserveTypeAliasesWithInvalidTypes
+            false,
+            // usedAttributes
+            ['lines' => true, 'indexes' => true],
+            // parseDoctrineAnnotations
+            false,
+            // textBetweenTagsBelongsToDescription, default to false, exists since 1.23.0
+            // should be can used for
+            // @param int $a
+            //     paramA description
+            true
```

Comment:
this exist since phpdoc-parser 1.23.0 that may be used for drupal use case

Reply:
@ondrejmirtes the spaced new line empty seems cause indentation gone, given the following code:

```php
class Foo extends FooBar
{
    /**
     * @param \DateTime $foo
     *
     * Storage here is Foo\Storage. Not \Storage.
     * @return Storage
     */
    public function bar($foo){}
}
```

it goes:

```diff
  class Foo extends FooBar
 {
     /**
-     * @param DateTime $foo
-     *
-     * Storage here is Foo\Storage. Not \Storage.
-     * @return Storage
-     */
+     * @param DateTime $foo 
+    
+    Storage here is Foo\Storage. Not \Storage.
+    * @return Storage
+    */
```

Reply:
@samsonasik Please report this properly in phpstan/phpdoc-parser, ideally with a failing test. I can't debug this through rector-src.

Reply:
@ondrejmirtes thank you, I will try when I have a chance to reproduce a failing test case on `phpstan/phpdoc-parser`, I will set to default `false` for now.

Reply:
@TomasVotruba I reverted to false for now :), marked as `@todo` until it has solution for above empty line use case https://github.com/rectorphp/rector-src/pull/4590/commits/2270001fc79c34bf9f760ddd6dc28dd358379fd9